src/bootstrap.php

```diff
@@ -65,6 +68,10 @@
         'profiler.cache_dir' => __DIR__.'/../var/cache/profiler',
         'profiler.mount_prefix' => '/_profiler',
     ]);
+    $app->get('/error', function () use ($app) {
+        $app['logger']->error('Simulating error');
```

Comment:
we can probably remove the explicit logging call here, since a log will be written by the exception capture?

Reply:
Well, the whole controller can go...

Reply:
Define only with `debug` on?

Reply:
Yeah, could be useful to test in different environments.

Reply:
Already in a `$app['debug']` block.